tonic/src/transport/endpoint.rs

```diff
@@ -213,6 +213,9 @@ pub struct ClientTlsConfig {
     domain: Option<String>,
     cert: Option<Certificate>,
     identity: Option<Identity>,
+    #[cfg(feature = "openssl-roots")]
```

Comment:
I think in this case you want `cfg(or(feature = "opnessl-roots", feature = "rustls-roots"))`